motoman_driver/include/motoman_driver/simple_message/motoman_read_single_io_reply.h

```diff
@@ -57,15 +57,19 @@ namespace io_ctrl_reply
 /**
  * \brief Enumeration of Read Single IO reply result codes.
  */
-namespace ReadSingleIOReplyResults
+namespace ReadSingleIOReplyResultCodes
 {
-enum ReadSingleIOReplyResult
+enum ReadSingleIOReplyResultCode
 {
-  FAILURE    = 0,
-  SUCCESS    = 1
+  SUCCESS               =    0,

```

Comment:
It seems MotoROS, prior to #369, returned `0` for failure instead of success.

@ted-miller: did you change that on purpose? Could be we made an incorrect assumption in this code here though.

Reply:
Prior to 369, the resultCode was being assigned with an enum of type SmReplyType. 
https://github.com/ros-industrial/motoman/blob/kinetic-devel/motoman_driver/MotoPlus/SimpleMessage.h#L95

I believe this was wrong and should have actually been using SmResultType, which has a different value for success.
https://github.com/ros-industrial/motoman/blob/kinetic-devel/motoman_driver/MotoPlus/SimpleMessage.h#L123

For other functions in the application, a status of '0' indicates success. Any other value is an error code.

Reply:
(But no, it wasn't on purpose. I just now discovered the different enum.)